Dark-contrast/Dark-contrast.qss

```diff
@@ -94,7 +94,7 @@ QToolBar  {
 }
 
 QMdiArea {
-    background-image: url(qss:images_dark-light/background_freecad_dark.svg);
+    background-image: url(qss:/../images_dark-light/background_freecad_dark.svg);
```

Comment:
I suppose that you tested it, and it works but this syntax seems really strange - this is one directory above the top one which is not a thing, why is it done like that?

Reply:
Functionality it's no difference, it was my Dos 3.1 heritage coming to the surface, the shortened path just doesn't look correct to my vintage, all back to shortened paths now.